mobile-widgets/themeinterface.h

```diff
@@ -33,7 +33,7 @@ class themeInterface : public QObject {
 
 	// Support
 	Q_PROPERTY(QString currentTheme MEMBER m_currentTheme WRITE set_currentTheme NOTIFY currentThemeChanged)
-	Q_PROPERTY(QString iconStyle MEMBER m_iconStyle CONSTANT)
+	Q_PROPERTY(QString iconStyle MEMBER m_iconStyle NOTIFY iconStyleChanged)
```

Comment:
this looks like it might be worth mentioning in the commit message. I assume something didn't work correctly because we forgot to add a notifier signal?

Reply:
Yes I assumed it was constant, but it changes when the theme changes.